docs/model/spec.rst

```diff
@@ -454,13 +454,15 @@ This model function is **required** if the ``password`` grant is used.
 
 **Arguments:**
 
-+------------+----------+---------------------------------------------------------------------+
-| Name       | Type     | Description                                                         |
-+============+==========+=====================================================================+
-| username   | String   | The username of the user to retrieve.                               |
-+------------+----------+---------------------------------------------------------------------+
-| password   | String   | The user's password.                                                |
-+------------+----------+---------------------------------------------------------------------+
++-------------------+----------+---------------------------------------------------------------------+
+| Name              | Type     | Description                                                         |
++===================+==========+=====================================================================+
+| username          | String   | The username of the user to retrieve.                               |
++-------------------+----------+---------------------------------------------------------------------+
+| password          | String   | The user's password.                                                |
++-------------------+----------+---------------------------------------------------------------------+
+| client (optional) | Client   | The user's password.                                                |
```

Comment:
The description still says "The user's password." Also I think in this docs we don't have "Client" defined as in TypeScript, it should rather be an object, right?

Reply:
Client is already defined in some other functions like `saveToken`.

Reply:
Fixed the typo